simple-ping-time-app.md

```diff
@@ -0,0 +1,189 @@
+
+# Build Simple Ping Time App With Livewire Polling
+![undraw_Time_management_re_tk5w](https://user-images.githubusercontent.com/22311928/173588230-da0ba3fc-4a02-4661-ad1f-812bbff3de4f.png)
+
+## Problem
+
+You are working on your next big side project, and there are several hosting alternatives. Speed is an essential factor for your application. Which hosting option has the best speed?
+
+Your approach to determining the speed of your application is measuring the ping time interval. The ping time interval measures from when your client(browser) sends a request to your server to when your server returns a response to the client. How can you measure the ping time interval in a Livewire application?
```

Comment:
I think what we are actually measuring here technically, is the average ping speed. The ping time interval is just the interval between pings which is set to 1 second and can be randomly changed. I don't know if this is me being pedantic lol.